src/Sentry.AspNetCore/SentryTunnelMiddleware.cs

```diff
@@ -98,10 +98,14 @@ public async Task InvokeAsync(HttpContext context, RequestDelegate next)
                     Method = HttpMethod.Post,
                     Content = new StreamContent(memoryStream),
                 };
+                var existingForwardedFor = context.Request.Headers["X-Forwarded-For"];
```

Comment:
question: In case of `clientIp` being `null`, do we want to add existing `"X-Forwarded-For"` to the `sentryRequest` regardless, or only when there is a `RemoteIpAddress` set?